interface/ceed-operator.c

```diff
@@ -689,7 +690,7 @@ int CeedOperatorSetField(CeedOperator op, const char *field_name, CeedElemRestri
     op->has_restriction = true;  // Restriction set, but num_elem may be 0
   }
   CeedCall(CeedBasisReferenceCopy(b, &(*op_field)->basis));
-  if (!op->num_qpts && b != CEED_BASIS_COLLOCATED) CeedCall(CeedOperatorSetNumQuadraturePoints(op, num_qpts));
+  if (op->num_qpts == 0) CeedCall(CeedOperatorSetNumQuadraturePoints(op, num_qpts));
```

Comment:
This change also means that the user should probably never directly call `CeedOperatorSetNumQuadraturePoints` themselves anymore.